proto/umee/leverage/v1/leverage.proto

```diff
@@ -111,10 +110,10 @@ message Token {
 
   // Liquidation Incentive determines the portion of bonus collateral of
   // a token type liquidators receive as a liquidation reward.
-  string liquidation_incentive = 9 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.nullable)   = false,
-    (gogoproto.moretags)   = "yaml:\"liquidation_incentive\""
+  // Value is in basis points; allowed values are in [0, 10000] range.
+  uint32 liquidation_incentive = 9 [
```

Comment:
We don't generally comment range information in our proto files (even though `Token.Validate` range restricts most fields.)